oat-sa/extension-tao-itemqti
# Feature/AUT-120/support parsing figcaption element

Related to: https://oat-sa.atlassian.net/browse/AUT-120

Depends on: 
- [x] https://github.com/oat-sa/tao-core/pull/3540
- [x] https://github.com/oat-sa/tao-item-runner-qti-fe/pull/289
- [x] https://github.com/oat-sa/extension-tao-testqti/pull/2272
- [x] https://github.com/oat-sa/lib-tao-qti/pull/117
- [x] https://github.com/oat-sa/qti-sdk/pull/313


<del>For local test need to use brach from qti-sdk, please run  `composer require "qtism/qtism:dev-feature/legacy/TR-1499/support_qti_figure_figcaption_elements as 0.25.2" -W`</del>

# How to test:

- create new passage
- add image to it
- add/edit image caption
- save
- check that it is rendered properly in preview, in items
- check export/re-import, delivery compilation, test runner
